list_of_names.py

```diff
@@ -0,0 +1,33 @@
+def namelist(names):
+    namesText = list(map(lambda n: n['name'], names))
+    
+    if len(namesText) == 0:
+        return ''
+    elif len(namesText) == 1:
+        return namesText[0]
+    
+    name = ', '.join(namesText[:-1])
+
+    return f"{name} & {namesText[-1]}"
+
+namelist([])
+
+def namelist(names):
```

Comment:
You've got two different solutions for the problem in the same file here, is this deliberate?

Reply:
Yes, this is deliberate.